src/main/scala/net/snowflake/spark/snowflake/SnowflakeJDBCWrapper.scala

```diff
@@ -199,9 +199,9 @@ private[snowflake] class JDBCWrapper {
            |""".stripMargin.filter(_ >= ' '))
     }
 
-    val snowflakeClientInfo = Utils.getClientInfoString()
-    log.info(snowflakeClientInfo)
-    System.setProperty("snowflake.client.info", snowflakeClientInfo)
+    // Important: Set "snowflake.client.info" is very important!
```

Comment:
Why is it important ?

Reply:
This setting will make the Snowflake GS to know the connection is from Spark Connector. In some special case, GS will do some special handling.